### DISCUSSION TITLE
Multiple Balloons

### DISCUSSION THREAD

**Discussion by DarrylBD99** [nathanhoad/godot_dialogue_manager#409] [General]
Not sure whether to put this in QnA or Ideas, so I just put it as general. Anyways would it be possible to use more than one type of balloon?

Example: One balloon for speech dialogue, another for item gained prompt, and so on.

**Comment by eisegesis-623** [comment]
Yep! You just have to make another balloon scene and call it instead of the normal one. One way to do this is to duplicate the show_dialogue_balloon() method in the Dialogue Manager and just change the scene in the new method.

**Comment by nathanhoad** [comment]
Adding to what @Paul-Griffin-1517-1 said, if you want different balloons to show on different events then you'll need to code your own method to instantiate the appropriate balloon and add it to the scene. The build-in one assumes that there's only one kind of the balloon by default.
